src/code/dialogs/checklist.js

```diff
@@ -47,7 +44,13 @@ const OperationChecklistDialog = WDialog.extend({
 
   _displayDialog: async function () {
     const operation = getSelectedOperation();
-    this.sortable = this.getListDialogContent(operation, 0, false); // defaults to sorting by op order
+    loadFaked(operation);
```

Comment:
isn't this too "automatic" for you?

Reply:
it is, this feature is controlled by the user settings (disabled by default)

Reply:
ah, ok